src/document/DocumentAnnotator.ts

```diff
@@ -66,7 +66,24 @@ export default class DocumentAnnotator extends BaseAnnotator {
             if (this.isFeatureEnabled('highlightText')) {
                 managers.add(new HighlightManager({ location: pageNumber, referenceEl: pageReferenceEl }));
             }
-            managers.add(new RegionManager({ location: pageNumber, referenceEl: pageReferenceEl }));
+            if (this.isFeatureEnabled('discoverability')) {
+                const canvasLayerEl = pageEl.querySelector('.canvasWrapper') as HTMLElement;
+                const referenceEl = canvasLayerEl || pageReferenceEl;
+
+                managers.add(
+                    new RegionManager({
+                        location: pageNumber,
+                        referenceEl,
+                    }),
+                );
+            } else {
+                managers.add(
```

Comment:
It doesn't seem like this needs to be duplicated. Can we use a ternary to define the final `referenceEl` value based on whether the feature is enabled?